tests/run_sssd_tests.yml

```
@@ -19,12 +19,12 @@
 - name: Verify that contents of sssd conf are correct
   assert:
     that:
-      - __tlog_conf_content.stdout.find('scope={{ tlog_scope_sssd }}')
+      - 'scope={{ tlog_scope_sssd }}' in __tlog_conf_content.stdout
       # yamllint disable-line rule:line-length
-      - __tlog_conf_content.stdout.find('users={{ tlog_users_sssd | join(', ') }}')
+      - 'users={{ tlog_users_sssd | join(", ") }}' in __tlog_conf_content.stdout
       # yamllint disable-line rule:line-length
-      - __tlog_conf_content.stdout.find('groups={{ tlog_groups_sssd | join(', ') }}')
+      - 'groups={{ tlog_groups_sssd | join(", ") }}' in __tlog_conf_content.stdout
       # yamllint disable-line rule:line-length
-      - __tlog_conf_content.stdout.find('exclude_users={{ tlog_exclude_users_sssd | join(', ') }}')
+      - 'exclude_users={{ tlog_exclude_users_sssd | join(", ") }}' in __tlog_conf_content.stdout
       # yamllint disable-line rule:line-length
-      - __tlog_conf_content.stdout.find('exclude_groups={{ tlog_exclude_groups_sssd | join(', ') }}')
+      - 'exclude_groups={{ tlog_exclude_groups_sssd | join(", ") }}' in __tlog_conf_content.stdout
```

```suggestion
      - 'exclude_groups={{ tlog_exclude_groups_sssd | join(", ") }}' in __tlog_conf_content.stdout
```
hmm - this would work better with an intermediate variable:

```yaml
vars:
  expected_exclude_groups: exclude_groups={{ tlog_exclude_groups_sssd | join(", ") }}
```

then use `- expected_exclude_groups in __tlog_conf_content.stdout`